src-backend/api/views.py

```diff
@@ -37,6 +39,25 @@ def get_queryset(self):
         user = self.request.user
         return models.Page.objects.filter(procedure__owner_id__exact=user.id)
 
+    @list_route(methods=['PATCH'])
+    def partial_bulk_update(self, request, *args, **kwargs):
```

Comment:
Partial so we don't have to pass back all the attributes of a model. Only really need the specific fields that we are updating. This will still support sending back all the attributes of a model.